saleor/graphql/product/types/products.py

```diff
@@ -189,11 +196,15 @@ class Meta:
 class ProductVariant(CountableDjangoObjectType, MetadataObjectType):
     quantity = graphene.Int(
         required=True,
-        description="Quantity of a product in the store's possession, "
+        description="DEPRECATED: Will be removed in favour of stock. "
```

Comment:
All of these deprecated fields are regular Graphene fields that provide `deprecation_reason` for the deprecation message. We should revert the original descriptions and use `deprecation_reason` instead. The message can be the same in all these cases: "This field will be removed in Saleor 2.11. Use the `stock` field instead.`